### ISSUE TITLE
ExportV1 Util Crashes When Attempting to Export

### ISSUE THREAD

**Issue by tjsiron** [Apicurio/apicurio-registry#1806]
Hello!

I'm trying to follow the migration path from 1.3 to 2.0 and exporting my schemas to a zip. The utility provided worked on a local version of apicurio's registry, but when I tried to point it at our developer instance it immediately breaks and reports the below error. I'm not a java developer by trade so I'm a little lost on what it's trying to do, but I'm guessing it's missing a dependency of some sort - I used the prebuilt jar from the migration documentation however, so I'm not sure how I'd go about fixing that. 

```
Unable to load provider class org.glassfish.jersey.internal.RuntimeDelegateImpl using custom classloader jdk.internal.loader.ClassLoaders$AppClassLoader trying again with current classloader.: java.lang.ClassNotFoundException: org.glassfish.jersey.internal.RuntimeDelegateImpl
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at javax.ws.rs.ext.FactoryFinder.newInstance(FactoryFinder.java:85)
        at javax.ws.rs.ext.FactoryFinder.find(FactoryFinder.java:210)
        at javax.ws.rs.ext.RuntimeDelegate.findDelegate(RuntimeDelegate.java:111)
        at javax.ws.rs.ext.RuntimeDelegate.getInstance(RuntimeDelegate.java:96)
        at javax.ws.rs.core.Response$ResponseBuilder.newInstance(Response.java:839)
        at javax.ws.rs.core.Response.status(Response.java:590)
        at javax.ws.rs.WebApplicationException.<init>(WebApplicationException.java:216)
        at javax.ws.rs.WebApplicationException.<init>(WebApplicationException.java:103)
        at io.apicurio.registry.client.request.RequestHandler$ResultCallback.onResponse(RequestHandler.java:38)
        at retrofit2.OkHttpCall$1.onResponse(OkHttpCall.java:161)
        at okhttp3.RealCall$AsyncCall.execute(RealCall.java:174)
        at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)

java.lang.RuntimeException: java.lang.ClassNotFoundException: org.glassfish.jersey.internal.RuntimeDelegateImpl
        at javax.ws.rs.ext.RuntimeDelegate.findDelegate(RuntimeDelegate.java:129)
        at javax.ws.rs.ext.RuntimeDelegate.getInstance(RuntimeDelegate.java:96)
        at javax.ws.rs.core.Response$ResponseBuilder.newInstance(Response.java:839)
        at javax.ws.rs.core.Response.status(Response.java:590)
        at javax.ws.rs.WebApplicationException.<init>(WebApplicationException.java:216)
        at javax.ws.rs.WebApplicationException.<init>(WebApplicationException.java:103)
        at io.apicurio.registry.client.request.RequestHandler$ResultCallback.onResponse(RequestHandler.java:38)
        at retrofit2.OkHttpCall$1.onResponse(OkHttpCall.java:161)
        at okhttp3.RealCall$AsyncCall.execute(RealCall.java:174)
        at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ClassNotFoundException: org.glassfish.jersey.internal.RuntimeDelegateImpl
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:315)
        at javax.ws.rs.ext.FactoryFinder.newInstance(FactoryFinder.java:93)
        at javax.ws.rs.ext.FactoryFinder.find(FactoryFinder.java:210)
        at javax.ws.rs.ext.RuntimeDelegate.findDelegate(RuntimeDelegate.java:111)
        ... 12 more
```

**Comment by EricWittmann** [comment]
Ugh.  JAX-RS provider classloading issue.  @famartinrh any ideas here?

@tjsiron Some questions:

1. What version of Java did you use to run the tool?
2. Did you use the same Java environment when you ran the tool each time?

**Comment by famartinrh** [comment]
if you haven't tried the latest released version of the export utility I would suggest trying it https://github.com/Apicurio/apicurio-registry/releases/tag/2.0.1.Final

**Comment by EricWittmann** [comment]
@tjsiron Any updates/progress on this?

**Comment by ajborley** [comment]
I can reproduce this when API calls to my 1.3.x Apicurio registry fail. For example, if I provide an incorrect path on my URL or  incorrect credentials. I have tried with various Java versions including Oracle and IBM JDKs. 

Testing it out I think it's just missing a JAX-RS implementation for when the 1.3.x Apicurio rest client creates a new instance of the JAX-RS WebApplicationException class: https://github.com/Apicurio/apicurio-registry/blob/1.3.x/rest-client/src/main/java/io/apicurio/registry/client/request/RequestHandler.java#L38

I have raised PR https://github.com/Apicurio/apicurio-registry/pull/2072 to provide the quarkus/resteasy JAX-RS implementation as that seems to fix the issue for me.
